try:
  print(x/0)
except:
  print('An exception occurred')
finally:
    print("you enter value")
